core/components/atoms/button/button.js

```diff
@@ -268,10 +268,10 @@ Button.Element = styled.button`
 Button.Text = styled.span`
   display: inline-block;
   vertical-align: middle;
-
+  line-height: normal;
   /* Sub-pixel position adjustment */
-  /* See: https://github.com/auth0/cosmos/pull/947 */
-  margin-top: 2px;
+  /* See: pulls 947 and 1045 */
```

Comment:
Can we include full URLs to the PRs? So in most editors you can `Cmd + Click` it to enter the link.

Reply:
```suggestion
  /* See: https://github.com/auth0/cosmos/pull/947 */
  /* See: https://github.com/auth0/cosmos/pull/1045 */
```